redislock_test.go

```diff
@@ -142,6 +142,61 @@ var _ = Describe("Client", func() {
 		Expect(numLocks).To(Equal(int32(1)))
 	})
 
+	It("should prevent multiple locks with linear retry (fuzzing)", func() {
+		numLocks := int32(0)
+		wg := new(sync.WaitGroup)
+		var cacheOpts = &redislock.Options{
```

Comment:
Hmmm, sharing the same options across multiple Obtain calls is not really appropriate, but OK